plugins/Speech/__init__.py

```diff
@@ -383,7 +427,7 @@ def OnButton(event):
             (sizer2, 0, wx.EXPAND|wx.ALL, 5),
         )
         ACV = wx.ALIGN_CENTER_VERTICAL
-        sizer3 = wx.FlexGridSizer(0, 2, 5, 5)
+        sizer3 = wx.FlexGridSizer(0, 0, 5, 5)
```

Comment:
ok this newest commit should be good to go. I found the error in win32com and did a monkey patch to correct it. I did pass along the information to Mark Hammond. but form looking at the support ticket history i do not think there will be a fix for a longgggg time